react/features/invite/components/info-dialog/web/PasswordForm.js

```diff
@@ -126,6 +133,7 @@ class PasswordForm extends Component<Props, State> {
                         autoFocus = { true }
                         className = 'info-password-input'
                         onChange = { this._onEnteredPasswordChange }
+                        placeholder = { placeHolderText }
                         spellCheck = { 'false' }
                         type = 'text'
```

Comment:
What do you think about using type = 'number' instead? That would defer the digit checking and even length check to the browser, but then you'd have to add css to hide the number wheel (up/down arrows) which maybe is a bigger pain than checking the entered value.